liquibase-core/src/main/java/liquibase/database/AbstractJdbcDatabase.java

```diff
@@ -494,7 +494,7 @@ protected boolean isDateTime(final String isoDate) {
     }
     
     protected boolean isTimestamp(final String isoDate) {
-        return isoDate.length() >= "yyyy-MM-ddThh:mm:ss.SSS".length();
+        return isoDate.length() >= "yyyy-MM-ddThh:mm:ss.F".length();
```

Comment:
See OracleDatabase l.261f., the value which came up from OracleIntegrationTest was like ...".1"